tests/RegressionTests_hera.intel.log

```diff
@@ -1,6139 +1,3 @@
-Fri Mar 19 15:56:08 UTC 2021
+Tue Mar 23 04:51:44 EDT 2021
 Start Regression test
 
-
-baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20210318/INTEL/fv3_control
```

Comment:
Is this an empty log file?

Reply:
Perhaps this was mistakenly committed?

Reply:
I don't think so. The numbers for Dusan's work on timing will change each RT, so those lines will always be changing from now on.